packages/aws-cdk/lib/api/garbage-collector.ts

```diff
@@ -0,0 +1,484 @@
+import * as cxapi from '@aws-cdk/cx-api';
+import { CloudFormation, S3 } from 'aws-sdk';
+import * as chalk from 'chalk';
+import { ISDK, Mode, SdkProvider } from './aws-auth';
+import { print } from '../logging';
+import { DEFAULT_TOOLKIT_STACK_NAME, ToolkitInfo } from './toolkit-info';
+
+// Must use a require() otherwise esbuild complains
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const pLimit: typeof import('p-limit') = require('p-limit');
+
+const ISOLATED_TAG = 'aws-cdk:isolated';
+const P_LIMIT = 50;
+const DAY = 24 * 60 * 60 * 1000; // Number of milliseconds in a day
```

Comment:
Not that it matters but do we really not have some kind of Duration.day() or something?

Reply:
same thing -- we're getting numbers so doesn't feel right to cast to something else